client/src/components/podcasts/PodcastForm.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react"
+import { Redirect } from "react-router-dom"
+import ErrorList from "../ErrorList.js"
+import translateServerErrors from "../../services/translateServerErrors.js"
+
+const PodcastForm = (props) => {
+  const [podcastRecord, setPodcastRecord] = useState({
+    name: "",
+    description: "",
+  })
+  const [errors, setErrors] = useState([])
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+
+  const handleInputChange = (event) => {
+    setPodcastRecord({
+      ...podcastRecord,
+      [event.currentTarget.name]: event.currentTarget.value,
+    })
+  }
+
+  const onSubmitHandler = (event) => {
+    event.preventDefault()
+    addPodcast(podcastRecord)
+    clearForm()
+  }
+
+  const clearForm = (event) => {
+    // event.preventDefault()
```

Comment:
hissss commented out code hisssss

I know WHY you did this though. Unfortunately, if you want a clearForm button, AND to use this method on line 24, you have to separate out the logic into another method

a method that runs lines 29-32, that you call in your onSubmitHandler and in your clearForm callback. Otherwise pressing this button will trigger a post request (having buttons in forms causes them to submit

The alternative is to move your button *outside* of the form tag, but style it to look like its a part of